streams/src/test/java/org/apache/kafka/streams/integration/ResetIntegrationTest.java

```diff
@@ -406,24 +361,6 @@ private void cleanGlobal(final String intermediateUserTopic) {
         Assert.assertEquals(0, exitCode);
     }
 
-    private void waitUntilUserTopicsAreDeleted() {
```

Comment:
Could we get rid of `zkUtils` as well in `assertInternalTopicsGotDeleted`, by using `userTopicExists` in that function?